coss/opensource_clubs/models.py

```diff
@@ -41,6 +41,24 @@ def get_first_image(self):
     ]
 
 
+class AboutPage(Page):
+    description = RichTextField(blank=True)
+    intro = RichTextField(blank=True)
+    why = RichTextField(blank=True)
```

Comment:
The text here should either be limited to a certain amount of words otherwise it breaks the UI.